File: Python/3or5div.py
print("program find sum of numbers divisible by 3 or 5 from the given input range by user")
sum=0
num1=int(input("Enter 1st number"))
num2=int(input("Enter 2nd number "))
list1=[]
for i in range(num1,num2):
    if (i%3==0 or i%5==0):
        sum=sum+i
        list1.append(i)
print("sum of numbers divisible by 3 or 5 : ",sum) 
print("number of elements divisible by 3 or 5 : ",len(list1))   
print(list1)    

File: C-language/3or5div.c
#include <stdio.h>

int main()
{
    int n, num1, num2, sum = 0, count = 0;
    printf("Enter 1st number : \n");
    scanf("%d", &num1);
    printf("Enter 2nd number : \n");
    scanf("%d", &num2);
    for (n = num1; n < num2; n++)
    {
        if (n % 3 == 0 || n % 5 == 0)
        {
            sum = sum + n;
            count++;
        }
    }
    printf("The sum of numbers : %d \n", sum);
    printf("The count of numbers : %d \n", count);
}